src/site/_includes/partials/site-header.njk

```diff
@@ -7,7 +7,7 @@
       <span class="tooltip__content" id="menu-button-toolip">Open menu</span>
     </button>
     <a href="/" class="site-header__brand brand">
-      {{ svg('../../images/logo.svg', {label: 'web.dev'}) }}
+      {{ svg('../../images/logo-without-devrel.svg', {label: 'web.dev'}) }}
```

Comment:
I think we should clean-up here. The logo with the "by Chrome DevRel" subline will never be used again, so remove it, rename `logo-without-devrel` to just `logo` and one ever needs to wonder why there is an unused one.